frontend_server/lib/server.dart

```diff
@@ -279,9 +296,10 @@ class _FrontendCompiler implements CompilerInterface {
     _outputStream.writeln('result $boundaryKey');
     await invalidateIfBootstrapping();
     if (filename != null) {
-      setEntrypointFilename(filename);
+      setMainSourceFilename(filename);
     }
-    final Program deltaProgram = await _generator.compile(entryPoint: _entryPoint);
+    final Program deltaProgram =
+        await _generator.compile(entryPoint: _mainSource);
```

Comment:
as you renamed _entryPoint to _mainSource this now looks weird.
Any other alternative names for newly introduced entryPoints? tfaEntryPoints?

Reply:
Thanks for taking a look.

I agree that there is still a little bit of disambiguation left.

Eventually I'd like to switch all precompilation to use new entry points format (or remove all entry points files and replace them with Dart annotations), so I would prefer not to use 'tfa' in their name (as they may be used in other places too). These are lists of entry points into Dart code from embedder/native code.

What do you think about renaming 'entryPoint' argument of IncrementalCompiler?

Reply:
That is fine too.